package.json

```diff
@@ -4,7 +4,8 @@
   "main": "dist/index.js",
   "scripts": {
     "build": "tsc",
-    "test": "karma start"
+    "test": "karma start",
+    "lint": "tslint -p . -t verbose"
```

Comment:
Sorry, I am not familiar with the package.json grammar. What's the usage of 'lint'? Do you need to update the README.md file?